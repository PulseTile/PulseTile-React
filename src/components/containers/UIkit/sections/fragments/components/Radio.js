import React, { Component } from 'react';

export default class Radio extends Component {

    state = {
        isRadioTest1: false,
        isRadioTest2: false,
        isRadioInlineChange1: false,
        isRadioInlineChange2: false,
        isRadioInlineChange3: false,
    };

    onRadioChange1 = () => {
        this.setState({
            isRadioTest1: true,
            isRadioTest2: false,
        });
    };

    onRadioChange2 = () => {
        this.setState({
            isRadioTest1: false,
            isRadioTest2: true,
        });
    };

    onRadioInlineChange1 = () => {
        this.setState({
            isRadioInlineChange1: true,
            isRadioInlineChange2: false,
            isRadioInlineChange3: false,
        });
    };

    onRadioInlineChange2 = () => {
        this.setState({
            isRadioInlineChange1: false,
            isRadioInlineChange2: true,
            isRadioInlineChange3: false,
        });
    };

    onRadioInlineChange3 = () => {
        this.setState({
            isRadioInlineChange1: false,
            isRadioInlineChange2: false,
            isRadioInlineChange3: true,
        });
    };

    render() {
        const { isRadioTest1, isRadioTest2, isRadioInlineChange1, isRadioInlineChange2, isRadioInlineChange3 } = this.state;
        return (
            <div id="radio" className="ui-section">
                <strong className="ui-title">Radio</strong>
                <div className="form-group-wrapper">
                    <form action="">
                        <div className="form-group">
                            <div className="input-holder">
                                <div className="input-holder">
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp" onClick={() => this.onRadioChange1()}>
                                            <input type="radio" id="radioTest1" name="radio-test" checked={isRadioTest1} />
                                            <label for="radioTest1"></label>
                                        </div>
                                        <label className="fcustominp-label">Radio Test 1</label>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className="form-group">
                            <div className="input-holder">
                                <div className="input-holder">
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp" onClick={() => this.onRadioChange2()}>
                                            <input type="radio" id="radioTest2" name="radio-test" checked={isRadioTest2} />
                                            <label for="radioTest2"></label>
                                        </div>
                                        <label className="fcustominp-label">Radio Test 2</label>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className="form-group">
                            <div className="input-holder">
                                <div className="input-holder">
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp-state disabled">
                                            <div className="fcustominp">
                                                <input type="radio" id="radioTest6" name="radio-test" disabled />
                                                <label for="radioTest6"></label>
                                            </div>
                                            <label className="fcustominp-label">Radio Test 3</label>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </form>
                    <div className="form-group">
                        <div className="input-holder">
                            <div className="input-holder">
                                <div className="wrap-fcustominps-inline">
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp" onClick={() => this.onRadioInlineChange1()}>
                                            <input type="radio" id="radioTest3" name="radio-test-inline" checked={isRadioInlineChange1} />
                                            <label for="radioTest3"></label>
                                        </div>
                                        <label className="fcustominp-label">Radio Test Inline 1</label>
                                    </div>
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp" onClick={() => this.onRadioInlineChange2()}>
                                            <input type="radio" id="radioTest4" name="radio-test-inline" checked={isRadioInlineChange2} />
                                            <label for="radioTest4"></label>
                                        </div>
                                        <label className="fcustominp-label">Radio Test Inline 2</label>
                                    </div>
                                    <div className="wrap-fcustominp">
                                        <div className="fcustominp" onClick={() => this.onRadioInlineChange3()}>
                                            <input type="radio" id="radioTest5" name="radio-test-inline" checked={isRadioInlineChange3} />
                                            <label for="radioTest5"></label>
                                        </div>
                                        <label className="fcustominp-label">Radio Test Inline 3</label>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
};
