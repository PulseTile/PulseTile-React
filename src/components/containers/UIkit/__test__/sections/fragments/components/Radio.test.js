import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Radio from '../../../../sections/fragments/components/Radio';

Enzyme.configure({ adapter: new Adapter() });

const RADIO_NUMBER = 6;

describe('Component <Radio />', () => {
  it('should renders Radio with props correctly', () => {
    const component = shallow(<Radio />);
    expect(component).toMatchSnapshot();
    for (let i = 0; i < RADIO_NUMBER; i++) {
      component.find('.fcustominp').at(i).simulate('click');
      expect(component).toMatchSnapshot();
    }
  });
});
