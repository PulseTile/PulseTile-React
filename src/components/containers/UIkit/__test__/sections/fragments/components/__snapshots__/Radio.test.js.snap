// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Radio /> should renders Radio with props correctly 1`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 2`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 3`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 4`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 5`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 6`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Radio /> should renders Radio with props correctly 7`] = `
<div
  className="ui-section"
  id="radio"
>
  <strong
    className="ui-title"
  >
    Radio
  </strong>
  <div
    className="form-group-wrapper"
  >
    <form
      action=""
    >
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest1"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest1"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 1
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest2"
                  name="radio-test"
                  type="radio"
                />
                <label
                  for="radioTest2"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test 2
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="form-group"
      >
        <div
          className="input-holder"
        >
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp-state disabled"
              >
                <div
                  className="fcustominp"
                >
                  <input
                    disabled={true}
                    id="radioTest6"
                    name="radio-test"
                    type="radio"
                  />
                  <label
                    for="radioTest6"
                  />
                </div>
                <label
                  className="fcustominp-label"
                >
                  Radio Test 3
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
    <div
      className="form-group"
    >
      <div
        className="input-holder"
      >
        <div
          className="input-holder"
        >
          <div
            className="wrap-fcustominps-inline"
          >
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest3"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest3"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 1
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={false}
                  id="radioTest4"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest4"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 2
              </label>
            </div>
            <div
              className="wrap-fcustominp"
            >
              <div
                className="fcustominp"
                onClick={[Function]}
              >
                <input
                  checked={true}
                  id="radioTest5"
                  name="radio-test-inline"
                  type="radio"
                />
                <label
                  for="radioTest5"
                />
              </div>
              <label
                className="fcustominp-label"
              >
                Radio Test Inline 3
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
